<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadDICOMSeriesQt</class>
 <widget class="QMainWindow" name="ReadDICOMSeriesQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICOM Series</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonOpenFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderSlices">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFolderName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>381</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSlicesNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>381</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSlicesNumberTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Slices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFolderNameTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder:</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
